docs/release-notes/8.0.37-29.md

```diff
@@ -0,0 +1,58 @@
+
+Improvements and bug fixes introduced by Oracle for MySQL 8.0.37 and included in Percona XtraDB Cluster are the following:
+
+This release enhances telemetry and provides a more comprehensive explanation of how the telemetry works, including new components, metrics files, and additional methods for disabling telemetry. Find more information in the [Telemetry on Percona XtraDB Cluster](../telemetry.md) document.
+
+Improvements and bug fixes provided by Oracle for MySQL 8.0.37 and included in Percona Server for MySQL are the following:
+
+* The redo log might not record a change in column order when using instant DDL, potentially leading to an inaccurate log replay during the recovery process. (Bug #35183686)
+
+* Enhanced management of buffers has been implemented in deleting tablespaces to prevent a possible assertion failure. (Bug #35676106, Bug #36343647)
+
+* Resolved an assertion failure associated with full-text indexes. (Bug #35836581)
+
+Find the complete list of bug fixes and changes in the [MySQL 8.0.37 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-37.html).
+
+## Bug fixes
+
+* [PXC-4173](https://perconadev.atlassian.net/browse/PXC-4173): Percona XtraDB Cluster node stoped when parallel replication workers were executing DDLs via async node.
+
+* [PXC-4313](https://perconadev.atlassian.net/browse/PXC-4313): Statements executed in RSU mode generated local GTID events.
+
+* [PXC-4363](https://perconadev.atlassian.net/browse/PXC-4363): Concurrent CREATE and DROP USER queries on different nodes resulted in permanent lock.
+
+* [PXC-4385](https://perconadev.atlassian.net/browse/PXC-4385): Running a TOI ALTER USER in one node and a SET PASSWORD in another node, resulted in statements hanging  and locking the affected nodes.
+
+* [PXC-4397](https://perconadev.atlassian.net/browse/PXC-4397): Both ALTER DATABASE and DROP TRIGGER hung when running these commands simultaneously.
+
+* [PXC-4401](https://perconadev.atlassian.net/browse/PXC-4401): There was duplicated GU_DBUG_SYNC_WAIT("before_replicate_sync") in galera::ReplicatorSMM::replicate().
+
+* [PXC-4403](https://perconadev.atlassian.net/browse/PXC-4403): Wrong field metadata caused assertion in protocol_classic.cc::store_string().
+
+* [PXC-4419](https://perconadev.atlassian.net/browse/PXC-4419): The concurrent drop of users on different cluster nodes resulted in a hanging node.
+
+* [PXC-4428](https://perconadev.atlassian.net/browse/PXC-4428): Garbd terminated SST script children after SST was completed.
+
+## Packaging notes
+
+Percona XtraDB Cluster 8.0.37-29 supports Ubuntu 24.04.
+
+
+## Useful links
+
+[Install Percona XtraDB Cluster](../install-index.md)
+
+The [Percona XtraDB Cluster GitHub location](https://github.com/percona/percona-xtradb-cluster)
+
+Download product binaries, packages, and tarballs at [Percona Product Downloads](https://www.percona.com/downloads)
```

Comment:
What about
downloads
install
and then GitHub repository

Reply:
reordered